modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -765,8 +880,11 @@ PERF_TEST_P_(BitwisePerfTest, TestPerformance)
     }
 
     // Comparison ////////////////////////////////////////////////////////////
-    EXPECT_EQ(0, cv::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
-    EXPECT_EQ(out_mat_gapi.size(), sz);
+    {
+        EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));
+        EXPECT_EQ(0, cv::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
+        EXPECT_EQ(out_mat_gapi.size(), sz);
```

Comment:
Please leave only one `EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));` and remove the rest.

Reply:
About  size check -  I suggested to keep it
https://github.com/opencv/opencv/pull/21150#discussion_r759034051
What do you think about it?

Reply:
Applied, thanks. I delete all unnecessary size comparison through the file